### PR TITLE
Tweaks to hs.tangent

### PR DESCRIPTION
**Pull request by latenitefilms**
- Removed `hs.tangent.interval`
- Added better debugging messages to socket callback

### CODE REVIEW

File: extensions/tangent/init.lua [hunk 1]
```diff
@@ -1265,7 +1260,7 @@ end
 --- Function
 --- Updates the Hub with a menu value.
 --- The Hub then updates the displays of any panels which are currently
----   showing the menu.
+--    showing the menu.
```
Comment (cmsj) [inline extensions/tangent/init.lua]: I don’t think this is intentional?

File: extensions/tangent/init.lua [hunk 2]
```diff
@@ -1061,7 +1056,7 @@ end
 --- Notes:
 ---  * Full documentation for the Tangent API can be downloaded [here](http://www.tangentwave.co.uk/download/developer-support-pack/).
 ---  * The callback function should expect 1 argument and should not return anything.
----  * The 1 argument will be a table, which can contain one or many commands. Each command is it's own table with the following contents:
+--   * The 1 argument will be a table, which can contain one or many commands. Each command is it's own table with the following contents:
```
Comment (cmsj) [inline extensions/tangent/init.lua]: I don’t think changing the `—` was intentional?

File: extensions/tangent/init.lua [hunk 3]
```diff
@@ -1695,6 +1690,47 @@ local MESSAGE_SIZE = 1
 -- Message Body.
 local MESSAGE_BODY = 2
 
+-- socketCallback(data, tag) -> none
```
Comment (cmsj) [inline extensions/tangent/init.lua]: `hs.tangent.socketCallback` here. Also I’m not sure what the reasoning is for exposing the callback - is it for people to use as their own Hs.socket callback?

**Comment by cmsj** [comment]
Added some comments in the code

File: extensions/tangent/init.lua (hunk 1 above)
Comment (latenitefilms) [reply extensions/tangent/init.lua]: Ah, no - not sure what happened there!

File: extensions/tangent/init.lua (hunk 2 above)
Comment (latenitefilms) [reply extensions/tangent/init.lua]: Ah, no - not sure what happened there!

File: extensions/tangent/init.lua (hunk 3 above)
Comment (latenitefilms) [reply extensions/tangent/init.lua]: It's not exposed - it's just in a local function.

**Comment by latenitefilms** [comment]
Fixed! Thanks @cmsj !

**Pull request merged**